### PR TITLE
Test open()

### PR DESCRIPTION
**Pull request by webmaster128**

### CODE REVIEW

File: contracts/swap/AtomicSwapEther.sol [hunk 1]
```diff
@@ -14,7 +14,7 @@ contract AtomicSwapEther {
     }
 
     enum States {
-        INVALID,
+        NON_EXISTING,
```
Comment (willclarktech) [inline contracts/swap/AtomicSwapEther.sol]: Suggest `NON_EXISTENT`

File: contracts/swap/AtomicSwapEther.sol [hunk 2]
```diff
@@ -27,8 +27,8 @@ contract AtomicSwapEther {
     event Expire(bytes32 _swapID);
     event Close(bytes32 _swapID, bytes _secretKey);
 
-    modifier onlyInvalidSwaps(bytes32 _swapID) {
-        require(swapStates[_swapID] == States.INVALID);
+    modifier onlyNonExistingSwaps(bytes32 _swapID) {
```
Comment (willclarktech) [inline contracts/swap/AtomicSwapEther.sol]: As above

File: contracts/swap/AtomicSwapEther.sol [hunk 3]
```diff
@@ -27,8 +27,8 @@ contract AtomicSwapEther {
...
+        require(swapStates[_swapID] == States.NON_EXISTING, "Swap ID must not exist");
```
Comment (willclarktech) [inline contracts/swap/AtomicSwapEther.sol]: Suggest "Swap ID already exists"

File: contracts/swap/AtomicSwapEther.sol (hunk 1 above)
Comment (webmaster128) [reply contracts/swap/AtomicSwapEther.sol]: 👍 thanks

**Pull request merged**